sentinelone/komand_sentinelone/connection/connection.py

```diff
@@ -142,22 +142,20 @@ def agents_support_action(self, action: str, agents_filter: str, module: str):
             "data": {"module": module}
         })
 
-    def get_threat_summary(self):
-        """ Output a summary of current threats in the system grouped by
-        mitigation level. """
+    def get_threat_summary(self, limit: int = 1000, offset: int = 0):
+        endpoint = f"threats?limit={limit}&skip={offset}"
```

Comment:
It looks like the "skip" query parameter can must be less than or equal to 1000...what would happen if a console has more than 1000 threats on it? Using the "nextCursor" response from the pagination object might be the best (if possible, I know that that object can be weird) to avoid this